invenio_oauthclient/contrib/cern.py

```diff
@@ -159,6 +160,9 @@
 REMOTE_APP_RESOURCE_SCHEMA = "http://schemas.xmlsoap.org/claims/"
 
 
+oauth_cern_response_received = signal('oauth-cern-response-received')
```

Comment:
@jirikuncar From ```handlers.py``` there is no access to get all the resources of the user. Should the methods ```account_info``` and ```account_setup``` return a dictionary with all the fields?